compiler/parser/src/parser.rs

```diff
@@ -23,6 +25,17 @@ pub fn parse_program(source: &str, source_path: &str) -> Result<ast::Suite, Pars
     })
 }
 
+/// Parse the token stream for a full python program.
+pub fn parse_program_tokens(
```

Comment:
I understand we'd better to add `parse_tokens` for some use case. But `parse_program_tokens` too?
This is a very specific use case which can be also covered by `parse_program_tokens`.

Reply:
Removed.